File: app/src/main/java/com/thatcakeid/zrytezene/DebugActivity.java
package com.thatcakeid.zrytezene;

import androidx.appcompat.app.AlertDialog;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import android.os.Bundle;
import android.view.View;

import com.thatcakeid.zrytezene.databinding.ActivityDebugBinding;

public class DebugActivity extends AppCompatActivity {

    private ActivityDebugBinding binding;

    private Toolbar toolbar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityDebugBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);

        toolbar = binding.toolbar;
        setSupportActionBar(toolbar);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        getSupportActionBar().setDisplayShowHomeEnabled(true);
        toolbar.setNavigationOnClickListener(v -> onBackPressed());

        AlertDialog.Builder alertDialog = new AlertDialog.Builder(this);
        alertDialog.setTitle("App crashed");
        alertDialog.setMessage(getIntent().getStringExtra("error"));
        alertDialog.setPositiveButton("Exit", (dialog, which) -> finish());
        alertDialog.create().show();

    }
}

File: app/src/main/java/com/thatcakeid/zrytezene/DebugActivity.kt
package com.thatcakeid.zrytezene

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AlertDialog
import androidx.appcompat.app.AppCompatActivity
import com.thatcakeid.zrytezene.databinding.ActivityDebugBinding

class DebugActivity : AppCompatActivity() {
    private val binding by lazy {
        ActivityDebugBinding.inflate(layoutInflater)
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val view: View = binding.root
        setContentView(view)

        val toolbar = binding.toolbar
        setSupportActionBar(toolbar)

        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        supportActionBar!!.setDisplayShowHomeEnabled(true)

        toolbar.setNavigationOnClickListener { onBackPressed() }

        AlertDialog.Builder(this)
            .setTitle("App crashed")
            .setMessage(intent.getStringExtra("error"))
            .setPositiveButton("Exit") { _, _ -> finish() }
            .create().show()
    }
}